src/server/game/Spells/SpellMgr.h

```diff
@@ -593,6 +593,14 @@ struct SpellInfoLoadHelper
     SpellTotemsEntry const* Totems = nullptr;
 };
 
+struct TC_GAME_API SpellTotemModel
+{
+    uint32 raceID;
```

Comment:
iirc we are using UpperCamelCase for structs, not sure though, kinda mixed in current tc code